### DISCUSSION TITLE
Hiding tab bar

### DISCUSSION THREAD

**Discussion by greenieves** [notaapp/talk#263] [Feature Requests]
Will there be an option to hide the tab bar on top? I really like a distraction free clean interface like iA writer.

**Comment by astoilkov** [comment]
We are considering it. Do you want it to auto-hide like in iA Writer or always be hidden?

Also (most likely in the next version), we will ship a simplified tab bar UX.

**Reply by greenieves** [reply]
Any updates on this? Sorry, I just really hate that the tab bar blocks an otherwise clean view of my notes

**Reply by astoilkov** [reply]
We haven't started working on this. I want to give an estimate so I'm more helpful to you but we have some other priorities we are working on right now.

**Comment by greenieves** [comment]
I would personally prefer a toggle to keep it hidden always
